### PR TITLE
Setup example feature

### PR DESCRIPTION
**Pull request by xPatiCiosek**
- Added WebDriverManager and JUnit dependencies
- Added Hooks class
- Added page objects
- Added example feature and steps

### CODE REVIEW

File: e-commerce_project/src/test/java/PageObjects/BasePage.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package PageObjects;
+
+import io.cucumber.core.internal.com.fasterxml.jackson.databind.ser.Serializers;
+import org.openqa.selenium.WebDriver;
+ abstract class BasePage {
+    WebDriver driver;
+
```
Comment (lina-aminov) [inline e-commerce_project/src/test/java/PageObjects/BasePage.java:7]: Inline comment

**Review by lina-aminov — APPROVED** [review]
Approved looks goot to me

**Review by odesanusi — APPROVED** [review]

**Review by HaiderB22 — APPROVED** [review]
Hooks class is fine!

**Review by jadjei19 — COMMENTED** [review]
Looks good Approved

**Review by jadjei19 — COMMENTED** [review]
Looks good

**Review by jadjei19 — APPROVED** [review]

**Pull request merged**